config/pulse.php

```diff
@@ -204,7 +204,7 @@
             'sample_rate' => env('PULSE_SLOW_REQUESTS_SAMPLE_RATE', 1),
             'threshold' => env('PULSE_SLOW_REQUESTS_THRESHOLD', 1000),
             'ignore' => [
-                '#^/pulse$#', // Pulse dashboard...
+                '#^/' . env('PULSE_PATH', 'pulse') . '$#', // Pulse dashboard...
```

Comment:
@rozsazoltan what do you think about extracting `'pulse'` string to the constant and use this constant in all places where needed default path? Because you found good point where we omit all places where PULSE_PATH is used

Reply:
@rozsazoltan and what if I declared `PULSE_PATH=/pulse-custom` in my env config, this regexp will work correctly ? because we will have to `//`

Reply:
> @rozsazoltan what do you think about extracting `'pulse'` string to the constant and use this constant in all places where needed default path? Because you found good point where we omit all places where PULSE_PATH is used

I was thinking about this as well, in the end, for the sake of a shorter code modification, I didn't highlight the `pulse` path, but we can do that if desired.

Reply:
I agree with you, but we need the opinion of the core team members here.